docs/development_suite/identity-and-access-management/group-management.md

```diff
@@ -0,0 +1,140 @@
+:::
+
+
+<div style={{display: 'flex', justifyContent: 'center'}}>
+  <div style={{display: 'flex', width: '600px'}}>
+
+![Edit group](./img/group-management/edit_group.png)
+
+  </div>
+</div>
+
+### Managing Group role at Company level
+
+Groups are shown into the [Identities portal](/development_suite/identity-and-access-management/overview.md#identities-portal) inside the Company Overview.
+As a result, the Company role of a group can be modified not only from the Groups section, but also from the Identities portal: to do so, simply click on the edit button for the desired group row and select the new role.
+
+
+<div style={{display: 'flex', justifyContent: 'center'}}>
+  <div style={{display: 'flex', width: '600px'}}>
+
+![Edit group role at Company level](./img/group-management/edit-group-role-at-company-level.png)
+
+  </div>
+</div>
+
+### Managing Group role at Project level
+
+Groups which have access to a Company are shown into the [Identities portal](/development_suite/identity-and-access-management/overview.md#identities-portal) inside the Project Settings Area of a Project belonging to that specific Company.
+Group role in the Project or any of the Project Runtime Environments can be modified. To do so, just open the editing dialog and select the proper role for the Project itself or for each Runtime Environment.
+
+
+<div style={{display: 'flex', justifyContent: 'center'}}>
+  <div style={{display: 'flex', width: '600px'}}>
+
+![Edit group role at Project level](./img/group-management/edit-group-role-at-project-level.png)
+
+  </div>
+</div>
+
+:::note
+Even though the Project Identities administration portal shows all the Company identities, this does not mean that all the identities have access to the Project, as this depends on their assigned role in the Company and how the permissions are inherited.
+
+For further information about permissions and role inheritance, check out the [Console Levels and Permission Management](/development_suite/identity-and-access-management/console-levels-and-permission-management.md) page.
+:::
+
+### Deleting a Group
+
+A group can be deleted by accessing its detail page and clicking on the *Delete* button at the end of the page and then confirming the action.
+
+:::caution
+Group deletion may result in loss of access to the Company for those group members who are not members of at least one other Company group or who do not have a specific individual Company role assigned.   
+
+In the case that a group member has access to the Company solely from its membership of the deleted group, the removal from that group will result in the loss of access to the Company for that user.  
```

Comment:
```suggestion
In the event that a member of a group has access to the Company solely by virtue of being a member of a group, the deletion of that group or the removal from it will result in the loss of access to the Company for that user.  
```